specification/app/resource-manager/Microsoft.App/ContainerApps/stable/2025-07-01/LogicAppsExtension.json

```diff
@@ -0,0 +1,525 @@
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Workflow information returned.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowEnvelope"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GET a workflow": {
+            "$ref": "./examples/LogicApps_GetWorkflow.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/providers/Microsoft.App/logicApps/{logicAppName}/listWorkflowsConnections": {
+      "post": {
+        "tags": [
+          "LogicApps"
+        ],
+        "summary": "Gets logic app's connections.",
+        "operationId": "LogicApps_ListWorkflowsConnections",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ContainerAppNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LogicAppNameParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Connections information returned.",
+            "schema": {
+              "$ref": "#/definitions/WorkflowEnvelope"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List the Workflows Configuration Connections": {
+            "$ref": "./examples/LogicApps_ListConnections.json"
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ContainerAppNameParameter": {
+      "name": "containerAppName",
+      "in": "path",
+      "description": "Name of the Container App.",
+      "required": true,
+      "type": "string",
+      "pattern": "^[-\\w\\._\\(\\)]+$"
+    },
+    "LogicAppNameParameter": {
+      "name": "logicAppName",
+      "in": "path",
+      "description": "Name of the Logic App.",
+      "required": true,
+      "type": "string",
+      "pattern": "^[-\\w\\._\\(\\)]+$"
```

Comment:
same as https://github.com/Azure/azure-rest-api-specs/pull/37658#discussion_r2373910239

Reply:
Here's fix, thank you

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/37699